.goreleaser.yml

```diff
@@ -0,0 +1,23 @@
+build: 
+  ldflags: -s -w -X main.Version={{.Version}}
+  binary: trawl
+  env: 
+    - CGO_ENABLED=0 
+  goos: 
+    - linux
+    - darwin
+    - windows
+  goarch: 
+    - amd64
+archive:
+  format: tar.gz
+snapshot: 
+  name_template: "{{.Commit}}"
+fpm:
```

Comment:
looks as though this needs to be updated - https://goreleaser.com/deprecations/#fpm

Reply:
@robphoenix done